frontend/src/components/App/App.js

```diff
@@ -8,48 +8,32 @@ import triggerAudioVisuals from '../../triggerAudioVisuals';
 import DataVisualiser from '../DataVisualiser/DataVisualiser.js';
 import { Routes, Route } from "react-router-dom";
 import processTubeData from '../../processTubeData';
-import allStations from '../../stations';
+// import allStations from '../../stations';
 import TIMEOUTS from '../../timeouts';
 import SideBarLeft from '../SideBarLeft/SideBarLeft';
 import Navbar from '../Navbar/Navbar';
 import Landing from '../Landing/Landing';
 import logo from '../../logo.svg';
 import * as Tone from 'tone';
+import fadeAllStations from '../../fadeAllStations';
 
-const dataBlockDuration = 30; // seconds between fetch from TFL
+const dataBlockDuration = 90; // seconds between fetch from TFL
 const lines = "bakerloo,central,circle,district,hammersmith-city,jubilee,metropolitan,northern,piccadilly,victoria,waterloo-city";
 const arrivals = []; // array to hold arrival elements, intialised w global scope
 let mainLooper;
 
 function App() {
   const [currentInstrument, setCurrentInstrument] = useState("orchestra")
-  const [visualiseEventsOnly, setVisualiseEventsOnly] = useState(true); // added for data visualiser
-  const [dataVisualiserKey, setDataVisualiserKey] = useState(0); // added for data visualiser
-  const [visualData, setVisualData] = useState([]); // added for data visualiser
+  const [visualiseEventsOnly, setVisualiseEventsOnly] = useState(true);
+  const [dataVisualiserKey, setDataVisualiserKey] = useState(0);
+  const [visualData, setVisualData] = useState([]);
   const [isPlaying, setIsPlaying] = useState(false);
-  const [arrivalEffectsToggle, setArrivalEffectsToggle] = useState(true); // added for data visualiser
+  const [arrivalEffectsToggle, setArrivalEffectsToggle] = useState(true);
   const [instruments, setInstruments] = useState(null);
   const [tapInVisible, setTapInVisible] = useState(true);
   const [muted, setMuted] = useState(false);
   const renderCount = useRef(1)
 
-  // const fadeElement = (elementId, state, setState) => {
```

Comment:
Ahhh .. that feeling when you delete a bunch of comments. It's like relaxing a freshly cleaned room